.gitignore

```
@@ -1 +1,3 @@
-*.exe
+*.out
```

achei vacilo ter tirado o .exe daí heim kkk, custa nada deixar kkk podemo colocar de volta? :sob: 
```suggestion
*.exe
*.out
```